testfiles/TF14/14.1-all-dna-1.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en-us">
+<head>
+<title>Link Purpose - no links (TC14.1-all-dna-1)</title>
+<html>
+<body>
+

```

Comment:
add where you want the code snippet to be

Reply:
I am not sure what you mean in your comment?